src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
I would share those as constants within the SDK or at least pull them from the global scope but import them via the SDK instead of just using them.

Even they are protected, these objects just hold simple strings as action, event mount ids. I really don't see that much value of using these objects. They don't really protect me from dispatching the same "CHANGE" event which is used in a different plugin as there is no validation within the plugin dispatcher and the values aren't unique in a way I can't accidently (or intentionally) use the same id.

What I most liked about your initial plugin design proposal was that there was nothing shared with plugins except the dispatcher. Everything else was part of a contract between Marathon UI and the plugins to use the same identifier for specific events as there is no way to enforce this.